frame/bridge/ethereum/relay/src/lib.rs

```diff
@@ -929,8 +930,11 @@ impl<T: Config> EthereumReceiptT<AccountId<T>, RingBalance<T>> for Module<T> {
 				mmr_proof.proof.to_vec(),
 				vec![(
 					ethereum_header.number,
-					array_bytes::dyn2array!(ethereum_header.hash.ok_or(<Error<T>>::HeaderHashInv)?, 32)
-						.into(),
+					array_bytes::dyn2array!(
+						ethereum_header.hash.ok_or(<Error<T>>::HeaderHashInv)?,
+						32
+					)
+					.into(),
 				)]
```

Comment:
i have run cargo-fmt in project root dir, i'm not sure it's the right thing

Reply:
Some code was reformated

Reply:
Okay, but made sure you're in the latest rust version.

Some rust-fmt got different behavior.